File: beecrowd -   1043 - Triângulo.py
a,b,c = map(float,input().split())
if ((b-c)<a<b+c) >0:
  if  ((a-c)<b<a+c) >0:
    if ((a-b)<c<a+b)>0:
      print('Perimetro = {:.1f}'.format(a+b+c))
else:
  at = ((a+b)*c)/2
  print("Area = {:.1f}".format(at))

File: beecrowd -   1043 - Triângulo.cpp
#include <iostream>
using namespace std;

int main() {
  double a,b,c,perimetro;
  cin >> a >> b >> c;
  cout.precision(1);
  if(a+b>c && a+c>b && b+c>a){
    perimetro = a+b+c;
    cout << fixed << "Perimetro = " << perimetro << endl;
  }
  else{
    cout << fixed << "Area = " << (a+b)*c/2 << endl;
  }

  return 0;
}
